roles/workstation/tasks/main.yml

```diff
@@ -0,0 +1,278 @@
+- name: Configure firewall
+  copy:
+    src: workstation.rules
+    dest: /etc/iptables.d/workstation.rules
+    owner: root
+    group: root
+    mode: 0640
+  notify:
+  - iptables
+
+- name: Install graphics libraries
```

Comment:
Would it be better to put some of these in [void-ansible-roles/xorg](https://github.com/void-ansible-roles/xorg)?  Maybe behind a variable option or two.